src/genie/libs/parser/nxos/show_track.py

```diff
@@ -0,0 +1,304 @@
+class ShowTrackSchema(MetaParser):
+    """ Schema for 'show track' """
+    schema = {
+        'track': {
+            Any(): {
+                Optional('type') : str,
+                Optional('instance'): str,
+                Optional('address'): str,
+                'state': str,
+                Optional('secs_remaining'): float,
+                Optional('sublist') : str,
+                Optional('subtrack') : str,
+                'change_count': int,
+                'last_change': str,
+                Optional('threshold_down'): str,
+                Optional('threshold_up'): str,
+                Optional('tracked_by'): {
+                    Optional(Any()): {   #increasing index 0, 1, 2, 3, ...
+                        Optional('name'): str,
+                        Optional('interface'): str,
+                        Optional('group_id'): str,
+                     }
+                },
+                Optional('track_list_members'):{
+                    Optional(Any()): {
+                        Optional('object_id') : str,
+                        Optional('weight') : str,	
+                        Optional('percentage') : str,
+                        Optional('obj_state') : str,
+                    }
+                },
+                Optional('delay_up_secs'): float,
+                Optional('delay_down_secs'): float,
+                Optional('delay_up_millisecs'): float,
+                Optional('delay_down_millisecs'): float,
+                Optional('VPN_vrf_Routing'): str,
+            },
+        },
+    }
+
+
+# =======================
+# Parser for 'show track'
+# =======================
+class ShowTrack(ShowTrackSchema):
+    """ Parser for 'show track' 'show track {id}'"""
+
+    cli_command = ['show track','show track {track_id}']
+
+    def cli(self, track_id = ' ', output=None):
+        if output is None:
+            if track_id:
+               cmd = self.cli_command[1].format(track_id=track_id)
+            else:
+                cmd = self.cli_command[0]
+
+            output = self.device.execute(cmd)
+
+        #Init vars
+        parsed_dict = {}
+       
+        #Track N 
+        p0 = re.compile(r'Track (?P<track_id>\d+)') 
+
+        # Interface Ethernet1/4 IP Routing
+        # IP Route 10.1.1.1/32 Reachability
+        # IPv6 Route 10:1::1:1/32 Reachability
+        # IP SLA 121 Reachability
+        p1 = re.compile(r'^(?P<type>IP Route|Interface|IP SLA|IPv6 Route|List)\s+((?P<address>[\d.]+(\/\d+))|(?P<name>([\d\w.:]+()\/\d+|loopback[\d]+|\d+)))*\s+((?P<parameter>[\w \-]+))')
+
+        # IP Routing is DOWN
+        # Line Protocol is delayed DOWN (20 secs remaining)
+        p2 = re.compile(r'^(?P<parameter>[\w ]+) +is +(?P<state>UP|DOWN|delayed DOWN|delayed UP)( \((?P<secs_remaining>\d+) secs remaining\))*')
+
+        # 1 changes, last change 00:01:04
+        p3 = re.compile(r'^(?P<change_count>\d+) +changes, +last +change'
+            r' +(?P<last_change>[\d\w:]+)')
```

Comment:
If wrapping line, please line up like this 
```suggestion
        p3 = re.compile(r'^(?P<change_count>\d+) +changes, +last +change'
                        r' +(?P<last_change>[\d\w:]+)')
```

Reply:
@LukasMcClelland made the necessary changes